### PR TITLE
Fixed Typos in Docs

### PR DESCRIPTION
**Pull request by 10kartik**
Fixed a few grammatical and sentence construction mistakes from docs.

### CODE REVIEW

File: docs/getting_started.rst [hunk 1]
```diff
@@ -274,8 +274,8 @@ pymemcache and fetch all the needed requirements at run. See the example below:
 You can instantiate all the classes and clients offered by pymemcache.
 
 Your client will remain open until you decide to close it or until you decide
-to quit your interpreter. It can allow you to see what's happen if your server
-is abruptly closed. Below is an by example.
+to quit your interpreter. It can allow you to see what happens if your server
+is abruptly closed. Below is a by-example.
```
Comment (jogo) [inline docs/getting_started.rst:278]: Out of curiosity can you point me to the grammar guideline for the dash in 'by-example' I haven't personally seen that one before.

**Review by jogo — APPROVED** [review]
Thank you for the cleanup! I have one very tiny question out of curiosity, will merge afterwards.

File: docs/getting_started.rst (hunk 1 above)
Comment (jparise) [inline docs/getting_started.rst:278]: I think this would be best as:

```suggestion
is abruptly closed. Below is an example.
```

Comment (10kartik) [reply docs/getting_started.rst:278]: yeah, sorry! There shouldn't have been '-' but 'by example' instead or 'an example' as suggested by @jparise below.

**Pull request merged**